tests/test_pagination.py

```diff
@@ -37,7 +37,8 @@ def testable_streams(self):
             {  # STREAMS NOT CURRENTY TESTABLE
                 'employees', # Requires production environment to create records
                 'modifier_lists',
-
+                'inventories',
```

Comment:
Why was this added to not-testable?

Reply:
oh maybe it wasn't supposed to be there, it must have been a merge mistake.